tests/autoyast/installation.pm

```diff
@@ -322,7 +322,7 @@ sub run {
 
     # Cannot verify second stage properly on s390x, so reconnect to already installed system
     if (is_s390x) {
-        reconnect_mgmt_console(timeout => 700, grub_timeout => 180);
+        reconnect_mgmt_console(timeout => 1400, grub_timeout => 360);
```

Comment:
we shouldn't increase the timeout in general, for x86_64 we might hide a bug, could you please add a condition here in another PR or file a ticket for it?